src/App.js

```diff
@@ -0,0 +1,8 @@
+function App() {
```

Comment:
Its good to define your standard early but usually newer react apps make use of arrow functions rather than function declarations, especially with JS

```suggestion
const App = () => {
```